### ISSUE TITLE
Graphical Modeling: Error "Character expected \n" with a comment line

### ISSUE THREAD

**Issue by tcquang** [gama-platform/gama.graphical.modeling#6]
Hello,

When adding a comment line ( // ) into the Gaml code box of an action or  a reflex, I got an error "Character expected \n" as in the figure attached. 
But it is OK with the comments by /*  */
I am using GAMA git version on Windows 10 64 bit.
Thank you!

Quang
![image](https://cloud.githubusercontent.com/assets/11825516/24236733/b810cd6e-0fd5-11e7-90ec-edfa38c40546.png)

**Comment by ptaillandier** [comment]
It seems to work for me. 
Maybe the error does not come from the commentary. Can try to delete the commentary and see if it is working without it?

**Comment by tcquang** [comment]
I just tested with new version of GM on GAMA continuous build version
It' still not working for me.
Please take a look on the diagram attached

Thank you!

[models.zip](https://github.com/gama-platform/gama.graphical.modeling/files/1646896/models.zip)



![image](https://user-images.githubusercontent.com/11825516/35156405-521109ee-fd63-11e7-91f8-42798fc51d5d.png)

**Comment by ptaillandier** [comment]
I commited a fix. It should work now (just update the plugin).

**Comment by tcquang** [comment]
Dear Patrick,
I just tested the new update of GM plugin, the issue seems to be not fixed.
1. In the fist case, I wrote a comment line above a code line, it didn't
know the variable var_a
//test comments
var_a <-var_a + var_a * 0.01;

2. In the seconds case, when I put a blank line, it didn't show errors but
it still shows red line on the top of the diagram. When I generated the
gaml code it show errors as in the 3rd image

Thank you for your help!
Best regards,
Quang

[image: Inline image 2]

[image: Inline image 1]
[image: Inline image 3]
